iree/turbine/kernel/wave/wave.py

```diff
@@ -576,8 +576,20 @@ def _trace_and_get_kernel_signature(
             partial(compute_shared_memory_usage, trace, options.kernel_launch_info),
         ]
 
+        pass_times = {}
         for p in graph_passes:
-            try_apply_pass(p, trace, print_ir_before, print_ir_after)
+            try_apply_pass(p, trace, print_ir_before, print_ir_after, pass_times)
+
+        if options.print_pass_times:
+            pass_times_list = sorted(
+                pass_times.items(), key=lambda x: x[1], reverse=True
+            )
+
+            import datetime
+
+            print(f"Pass times:")
+            for k, v in pass_times_list:
+                print(f"    {k}: {datetime.timedelta(seconds=v)}")
```

Comment:
The output is a little hard to read. Can we just print seconds (also add unit in output)? And also remove date time then?